ReverseTicTacToe/UI/ConsoleUI.cs

```diff
@@ -33,17 +29,145 @@ public  void Init()
         }
         public  void Run()
         {
+            
             WriteLine("Welcome to Reverse Tic Tac Toe!" + Environment.NewLine);
-            getBoardSizeFromUser();
-            getGameTypeFromUser();
-            m_Game = new Game(m_BoardSize, m_GameType);
-            while(true)
+            int BoardSize =  getBoardSizeFromUser();
+            Game.eGameTypes GameType = getGameTypeFromUser();
+            m_Game = new Game(BoardSize, GameType);
+           // while(true)
+            //{
+            printGameBoard();
+            playTurn();
+            ReadLine();
+            
+            //}
+        }
+
+        private void playTurn()
+        {
+            int row = 0;
+            int col = 0;
```

Comment:
if row and col are output paramterers, i think there's no need to initialize them. The function "turnInputIsValid" will be forced to do that i think

Reply:
I init them to 0 because in case of Computer turn it don't enter the If statement and does not got to this function